src/uwtools/tests/fixtures/include_files.sh

```diff
@@ -1,3 +1,3 @@
-salad_include=!INCLUDE [./fruit_config.sh]
+salad_include="!INCLUDE [./fruit_config.sh]"
```

Comment:
The `!INCLUDE` value must be quoted so that it is valid shell code. See `include_files.nml` for a parallel Fortran-namelist case.